WordPressVIPMinimum/Tests/Filters/RestrictedFilterUnitTest.inc

```diff
@@ -1,5 +1,15 @@
 <?php
 
-add_filter( 'test_filter', 'good_example_function' ); 	// this one shouldn't trigger anything
+add_filter( 'test_filter', 'good_example_function' ); // Ok.
 
-add_filter( 'upload_mimes', 'bad_example_function' );
\ No newline at end of file
+// warnings.
+add_filter( 'upload_mimes', 'bad_example_function' );
+add_filter('upload_mimes' ,'bad_example_function');
+add_filter(  'upload_mimes','bad_example_function');
+add_filter( "upload_mimes" ,'bad_example_function');
+add_filter( 'upLoad_mimeS' ,'bad_example_function');
+add_filter( 'upload_' . 'mimes' ,'bad_example_function');
+add_filter( 'upl' . 'oad_' . 'mimes' ,'bad_example_function');
+add_filter( 'upload_mimes', function() {
+	// do stuff.	
+});
```

Comment:
Not seeing any tests for dynamic hook names?

I guess someone would definitely be actively trying to game the system if they were using this, so it could always be added later if necessary.

Reply:
Actually, since these are tokens, it may not be possible parse the variable value and see if variable + string = upload_mimes.